__init__.py

```diff
@@ -11,11 +12,12 @@
 _LOGGER = logging.getLogger(__name__)
 
 DOMAIN = 'spotcast'
-CONF_DEVICE_NAME = 'device_name'
+CONF_DEVICE_NAME = 'entity_id'
```

Comment:
Oh, replaceing the device_name is not a smart idea because people are using it. 
Suggestion is to change the schema from 
```
vol.Required(CONF_DEVICE_NAME): cv.string,
```
to
````
vol.Optional(CONF_DEVICE_NAME): cv.string,
vol.Optional(CONF_ENTITITY_ID): cv.string,
```
Then do the logic to check which one is `not None` on line line 90

Could you also create 1 PR with the services file included? Right now I'm pretty busy so I can't help out...

Reply:
You can also use cv.deprecated to migrate from one to the other (and automatically tell users to change).